djcelery/management/base.py

```diff
@@ -22,34 +22,45 @@ def patch_thread_ident():
     # -- patch taken from gunicorn
     if getattr(patch_thread_ident, 'called', False):
         return
-    try:
-        from django.db.backends import BaseDatabaseWrapper, DatabaseError
 
-        if 'validate_thread_sharing' in BaseDatabaseWrapper.__dict__:
+    try:
+        # django >= 1.8
+        from django.db.backends.base.base import (BaseDatabaseWrapper,
+                                                  DatabaseError)
+    except ImportError:
+        try:
+            # django < 1.8
+            from django.db.backends import BaseDatabaseWrapper, DatabaseError
+        except ImportError:
+            return
+
+    if 'validate_thread_sharing' in BaseDatabaseWrapper.__dict__:
+        try:
             import thread
```

Comment:
This import doesn't work on Python 3. You should also try to import `_thread` for this patch to work on py3.